taui/src/components/geocoder.js

```diff
@@ -133,12 +133,25 @@ export default class Geocoder extends Component<Props> {
     }
   }
 
+  wrapperStyle = {
```

Comment:
These select styles could be moved to `constants.js`, so they aren't repeated in multiple files.